libcob/coblocal.h

```diff
@@ -449,6 +449,29 @@ COB_HIDDEN const char	*cob_get_last_exception_name	(void);
 COB_EXPIMP void		cob_field_to_string	(const cob_field *, void *,
 						 const size_t);
 COB_HIDDEN void		cob_parameter_check	(const char *, const int);
+COB_HIDDEN char*        cob_get_strerror (void);
+
+COB_HIDDEN JNIEnv*	cob_create_vm					();
+COB_HIDDEN void	cob_handle_error		(JavaVM* jvm, char* methodSig);
+COB_HIDDEN char* cob_gen_method_sig		(const char** paramType, int paramCount, const char** returnType);
+
+COB_HIDDEN void cob_lookup_static_method		(JNIEnv* env, JavaVM* jvm, const char *className, const char *methodName, 
+                              const char *methodSig, const char *returnType, const char** paramTypes, int paramCount);
+
+COB_HIDDEN void cob_static_method		(JNIEnv* env, JavaVM* jvm, jclass cls, jmethodID mid);
+COB_HIDDEN void cob_call_java_static_method		(JNIEnv *env, JavaVM *jvm, const char *className, const char *methodName, jobject obj, jstring input);
+
+enum cob_case_modifier {
+	CCM_NONE,
+	CCM_LOWER,
+	CCM_UPPER,
+	CCM_LOWER_LOCALE,
+	CCM_UPPER_LOCALE
+};
+COB_HIDDEN unsigned char	cob_toupper (const unsigned char);
+COB_HIDDEN unsigned char	cob_tolower (const unsigned char);
+COB_HIDDEN int		cob_field_to_string	(const cob_field *, void *,
+						 const size_t, const enum cob_case_modifier target_case);
```

Comment:
something has gone wrong here - see the same content directly above

Reply:
I do not see any duplicates in the file?

Reply:
cob_field_to_string is duplicate, cob_tolower/cob_toupper/cob_get_strerror are shown to be added while not related to this PR

Reply:
I might have changed the line number causing get strerror and the enum to appear in the PR